site/en/docs/privacy-sandbox/glossary/index.md

```diff
@@ -124,7 +132,7 @@ if something is missing!
 
 {% Partial 'privacy-sandbox/glossary-entries/federated-identity.njk' %}
 
-## Fenced frame
+## Fenced frame {: #fenced-framee }
```

Comment:
The auto-ID for this entry would be `#fenced-frame`, so there's no need to specify. Specification is important for shortened / other unique identifiers.

Reply:
Ok I thought you preferred we add them anyway

Reply:
Removed the repetitive anchors

Reply:
exactly. That was my typo. grrr :)